<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>formRapor</class>
 <widget class="QDialog" name="formRapor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Rapor Seçenekleri</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="simgeler.qrc" >:/new/prefix1/icons/date.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Rapor Seçenekleri</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="1" >
       <widget class="QDateEdit" name="dateMonth" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="displayFormat" >
         <string>MM.yyyy</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QRadioButton" name="radioMonthly" >
        <property name="text" >
         <string>Aylık Rapor</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QDateEdit" name="dateDay" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="displayFormat" >
         <string>dd.MM.yyyy</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QRadioButton" name="radioDaily" >
        <property name="text" >
         <string>Günlük Rapor</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>radioDaily</tabstop>
  <tabstop>dateDay</tabstop>
  <tabstop>radioMonthly</tabstop>
  <tabstop>dateMonth</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="simgeler.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>formRapor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioDaily</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateDay</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>93</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel" >
     <x>246</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioMonthly</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateMonth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>91</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel" >
     <x>241</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
